Resources/public/js/fragmentList.jquery.js

```diff
@@ -0,0 +1,337 @@
+                    // Append the new fragment
+                    self.added(html);
+                }
+            });
+
+            return false;
+        },
+
+        /**
+         * A new fragment was added
+         */
+        added: function(node) {
+            var $node = $(node),
+                id = $node.attr(this.options.formIdentifier),
+                data = $node.data('formdata');
+
+            // Update form list
+            this.setFormListElement();
+            // Append the new fragment
+            this.appendFragment(id, data, true);
+            // Reorder
+            this.resetOrder();
+        },
+
+        /**
+         * Remove an item (cross link clicked)
+         */
+        remove: function(e) {
+            e.preventDefault();
+            e.stopPropagation();
+
+            var $target = $(e.currentTarget),
+                $frag = $target.closest('[data-fragment]'),
+                id = $frag.data('fragId'),
+                $form = this.getFormByFragmentId(id),
+                $cb = $form.find(this.options.formRemoveName);
+
+            // Remove a persisted fragment (just check delete checkbox)
+            if($form.data('formTmp')) {
+                // Check the delete checkbox
+                $cb.prop('checked', true).attr('checked', true);
+                console.log($form.find(this.options.formRemoveName));
+
+                // Hide form block
+                $form.hide();
+            // Remove a new fragment (not already saved)
+            } else {
+                $form.remove();
+                this.setFormListElement();
+            }
+
+            // Remove list element
+            $frag.remove();
+
+            // Removed the last element ?
+            if(!this.$list.children().length) {
+                this.$activeFragment = null;
+            // Removed the current visible item ?
+            } else if(id === this.$activeFragment.data('fragId')) {
+                this.setActive(this.$list.children(':first-child'));
+            }
+
+            // Reorder
+            this.resetOrder();
+
+        },
+
+        /**
+         * Set an item as active
+         */
+        setActiveOnClick: function(e) {
+            var $target = $(e.currentTarget);
+
+            // Before adding, be sure the current fragment is valid
+            if(this.checkValidity()) this.setActive($target);
+        },
+
+        /**
+         * Get form matching to a fragment
+         */
+        getFormByFragmentId: function(fragId) {
+            var self = this,
+                found = false;
+
+            this.$formsList.each(function() {
+                var $this = $(this);
+
+                if(fragId === $this.attr(self.options.formIdentifier)) {
+                    found = $this;
+                }
+            });
+
+            return found;
+        },
+
+        /**
+         * Set an item as active
+         */
+        setActive: function($target) {
```

Comment:
No idea, it's more a joke than other. :wink: 

We need a JS expert here!

@ju1ius? :heart_decoration: